### PR TITLE
Semantic service's type endpoint returns the correct object id

### PR DESCRIPTION
**Pull request by ywkim312**

### CODE REVIEW

File: server/semantics-service/src/main/java/edu/illinois/ncsa/incore/service/semantics/controllers/TypeController.java [hunk 1]
```diff
@@ -109,6 +109,13 @@ public Response listTypes(
 
         List<Document> typeList = this.typeDAO.getTypes();
 
+        // hackish way of adding "id" field
```
Comment (navarroc) [inline server/semantics-service/src/main/java/edu/illinois/ncsa/incore/service/semantics/controllers/TypeController.java:112]: I don't think we need to do this here. The listTypes returns all the type "names", no objects, right @longshuicy ?

Comment (longshuicy) [reply server/semantics-service/src/main/java/edu/illinois/ncsa/incore/service/semantics/controllers/TypeController.java:112]: If set "detail" flag to true it returns all the objects :-(
We might want to revisit this design at some point...

**Review by longshuicy — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,6 +16,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Geoserver connection library has been removed and new connection object has been added [#190](https://github.com/IN-CORE/incore-services/issues/190)
 - Semantics should display description instead of title [#206](https://github.com/IN-CORE/incore-services/issues/206)
 
+### Fixed
+- Semantic services endpoint returns correct ID [#206](https://github.com/IN-CORE/incore-services/issues/206)
```
Comment (navarroc) [inline CHANGELOG.md:20]: This should be issue 227

Comment (longshuicy) [reply CHANGELOG.md:20]: haha i'll update that for you @ywkim312 . I'm in a hurry!

Comment (ywkim312) [reply CHANGELOG.md]: Updated

**Review by navarroc — APPROVED** [review]

**Pull request merged**